src/components/wizards/import-vm/ImportVmWizard.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { Form, FormControl, Col, DropdownButton, MenuItem, FormGroup, ControlLabel, Wizard } from 'patternfly-react';
+
+export class ImportVmWizard extends React.Component {
+
+    state = {
+      activeStepIndex: 0
+    }
+  
+    wizardStepsImportVM = [
+      {
+        title: 'Basic Information',
+        render: () => <Form horizontal>
+          <FormGroup>
+            <Col sm={3} className="text-right">
+              <ControlLabel className="required-pf">Import Source Type</ControlLabel>
+            </Col>
+            <Col sm={4}>
```

Comment:
Noticed this field isn't the same length as the rest of them in the list. Is it possible for them to align?

Reply:
length is the same now. The Import wizard is just a sketch, it has no functionality yet. Lets not spent any more time.

Reply:
> length is the same now. The Import wizard is just a sketch, it has no functionality yet. Lets not spent any more time.

Understood. Thanks!!